### DISCUSSION TITLE
Question on image display issue

### DISCUSSION THREAD

**Discussion by geniza** [CollectionBuilder/collectionbuilder.github.io#55] [Q&A] [answered]
Before I get to my question, firstly, thanks for developing this platform and all of the instructional materials! I have learned so much already in just a couple weeks. I'm using CollectionBuilder in order to develop a paleographical glossary for currency-related terms and have just managed to upload draft metadata and their corresponding objects. The issue I have encountered lies in the image display on my GitHub Pages site. Although I can download the .jpg or .pdf objects as standalone files when I "View Full Record" they are not displaying directly on the site: https://geniza.github.io/paleographicalglossary/browse.html
![Screen Shot 2022-06-06 at 6 19 26 PM](https://user-images.githubusercontent.com/106448975/172258833-578fbcf7-11d6-42aa-8d8a-52aef26598e9.png)

**Comment by evanwill** [comment, marked as answer]
@geniza thanks for reaching out, and glad you are enjoying learning!

CollectionBuilder-GH switches out the display based on the "format" field in your metadata. Your current metadata CSV does not have a "format" field included--so it is falling back to assuming all your items are some unknown object type! To get the images to display correctly in your collection, all you will need to do is add a "format" column to your metadata spreadsheet--here is the [documentation about "format" field](https://collectionbuilder.github.io/cb-docs/docs/metadata/gh_metadata/#format).

It looks like you were trying both ".jpg" and ".pdf" versions of your objects--> I think I would go with the ".jpg" version (as in the file "coinsofcairogenizabackendtest.csv"). Then add a column named `format`, and since all items are ".jpg", they will all have the format value of `image/jpeg`. Once the format is in your metadata, the collection features and item pages should work as expected!

Let me know how it goes!

**Reply by geniza** [reply]
This was so helpful, that fixed nearly everything. The only issue left is a few spelling inconsistencies between the metadata and the .jpg files– which will be easy to fix. Thanks again!
